source/applications/initializers.md

```diff
@@ -44,13 +44,17 @@ export default {
 ```
 
 ## Application Instance Initializers
+Application instance initializers are currently unavailable in Ember CLI,
```

Comment:
A slight rewording to avoid inferring that instance-initializers cannot be used with ember-cli:

> An application instance initializer generator is not currently available in Ember CLI,